frame/staking/src/lib.rs

```diff
@@ -580,6 +673,8 @@ pub trait SessionInterface<AccountId>: frame_system::Trait {
 	fn validators() -> Vec<AccountId>;
 	/// Prune historical session tries up to but not including the given index.
 	fn prune_historical_up_to(up_to: SessionIndex);
+	/// Current session index.
+	fn current_index() -> SessionIndex;
```

Comment:
doesn't seem to be used anymore after the merge
```suggestion
```

Reply:
Yeah great